.github/pr_template_standard.md

```diff
@@ -0,0 +1,39 @@
+## Description
+
+Please include a brief summary of the change.
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
```

Comment:
Should we add a section for the 'chore' type? I have seen a few of those in the commit history

Reply:
Good call thank you 👍